enterprise/backend/src/metabase_enterprise/cache/strategies.clj

```diff
@@ -28,64 +26,52 @@
     [:ttl      [:map {:closed true}
                 [:type [:= :ttl]]
                 [:multiplier ms/PositiveInt]
-                [:min_duration_ms ms/IntGreaterThanOrEqualToZero]]]
+                [:min_duration_ms ms/IntGreaterThanOrEqualToZero]
+                ^:internal
```

Comment:
what does this internal metadata do?

Reply:
This schema is reused in API (for PUT requests for example), and we're not waiting for this field to come from a client.